### ISSUE TITLE
loss 异常

### ISSUE THREAD

**Issue by winnerahao** [urgent-challenge/urgent2026_challenge_track1#1]
直接使用官方simulation合成的数据，以及官方share的参数运行bsrnn的训练，存在loss数值非常大且不收敛问题，请问官方bsrnn的训练loss在什么范围内？

**Comment by winnerahao** [comment]
另外，想问下bsrnn训练和bsrnn-flow两个模型训练的GPU内存有多大？24G好像训不起来呢，OOM

**Comment by LiChenda** [comment]
Hello, by default, the `reduction=sum` in `MultiResL1SpecLoss`, so the value is indeed very large, which is normal. In addition, the baseline was trained with 80 GB of GPU memory. You can appropriately reduce the batch size.

**Comment by winnerahao** [comment]
Thanks! Thanks very much!!

**Comment by takahashir986** [comment]
Hi, thank you for organizing the challenge!

I also have a question regarding the baseline models:
How many epochs/iterations did you train on both BSRNN and BSRNN-Flow? I tried to find the info in the [paper](https://arxiv.org/pdf/2506.23859), but could not find it.

Thank you!

**Comment by LiChenda** [comment]
> Hi, thank you for organizing the challenge!
> 
> I also have a question regarding the baseline models: How many epochs/iterations did you train on both BSRNN and BSRNN-Flow? I tried to find the info in the [paper](https://arxiv.org/pdf/2506.23859), but could not find it.
> 
> Thank you!

Hi, @takahashir986 , the baseline was trained for a maximum of 80 epochs, and the best validation loss checkpoint was chosen.

**Comment by xczhu98** [comment]
@LiChenda @winnerahao @takahashir986  你好，我使用官方配置和官方筛选的数据进行训练，loss非常大，而且训练的非常慢，使用的是8卡A800。这样的loss和速度正常吗？

Epoch 20:  92%|▉| 12548/13631 [3:48:45<19:44,  0.91it/s, v_num=0, train_loss=6.72e+4, train_sisnr=9.620, train_sisnr_16000=7.740, trNaN in loss has been decected, skip
Epoch 21:  37%|▎| 5027/13631 [1:33:33<2:40:08,  0.90it/s, v_num=0, train_loss=7.15e+4, train_sisnr=11.50, train_sisnr_16000=11.80, tNaN in loss has been decected, skip
Epoch 22:  81%|▊| 11027/13631 [3:29:18<49:25,  0.88it/s, v_num=0, train_loss=8.49e+4, train_sisnr=7.700, train_sisnr_16000=11.90, trNaN in loss has been decected, skip
Epoch 22:  83%|▊| 11381/13631 [3:35:05<42:31,  0.88it/s, v_num=0, train_loss=8.58e+4, train_sisnr=9.900, train_sisnr_16000=-2.63, trNaN in loss has been decected, skip
Epoch 23:   5%| | 727/13631 [11:22<3:21:54,  1.07it/s, v_num=0, train_loss=8.08e+4, train_sisnr=9.420, train_sisnr_16000=11.80, traiNaN in loss has been decected, skip
